components/decimal/src/format.rs

```diff
@@ -23,9 +23,10 @@ pub struct FormattedFixedDecimal<'l> {
 impl<'l> FormattedFixedDecimal<'l> {
     fn get_affixes(&self) -> Option<&AffixesV1> {
         match self.value.sign() {
-            Sign::Negative => Some(&self.symbols.minus_sign_affixes),
             Sign::None => None,
+            Sign::Negative => Some(&self.symbols.minus_sign_affixes),
             Sign::Positive => Some(&self.symbols.plus_sign_affixes),
+            _ => None, // This should not happen, because all the cases are covered.
```

Comment:
Should this be `unreachable!`? This will be hit when a new `Sign` variant is added, in which case we don't want to silently return `None`.

Reply:
good idea, I have changed it